### PR TITLE
Refactor create_seqinfo tiny bit to avoid duplication and add logging; and in tests to reuse list of dicom paths

### PR DESCRIPTION
**Pull request by yarikoptic**
Follow up to
- #753 
triggered by desire to avoid "growing" duplication and wanting to avoid call outs to `wrapper_from_file` at the test module import time.

### CODE REVIEW

File: heudiconv/tests/test_dicoms.py [hunk 1]
```diff
@@ -180,26 +180,17 @@ def test_get_datetime_from_dcm_wo_dt() -> None:
...
-]
-
-
-@pytest.mark.parametrize("mw,series_files,series_id", dicom_test_data)
+@pytest.mark.parametrize("dcmfile", TEST_DICOM_PATHS)
 def test_create_seqinfo(
-    mw: dw.Wrapper,
-    series_files: list[str],
-    series_id: str,
+    dcmfile,
```
Comment (bpinsard) [inline heudiconv/tests/test_dicoms.py:185]:
```suggestion
    dcmfile: str,
```

File: heudiconv/tests/test_dicoms.py [hunk 2]
```diff
@@ -180,26 +180,17 @@ def test_get_datetime_from_dcm_wo_dt() -> None:
...
-
+    mw = dw.wrapper_from_file(dcmfile)
+    seqinfo = create_seqinfo(mw, [dcmfile], op.basename(dcmfile))
+    assert seqinfo.sequence_name
 
-def test_get_reproducible_int() -> None:
-    dcmfile = op.join(TESTS_DATA_PATH, "phantom.dcm")
 
+@pytest.mark.parametrize("dcmfile", TEST_DICOM_PATHS)
+def test_get_reproducible_int(dcmfile) -> None:
```
Comment (bpinsard) [inline heudiconv/tests/test_dicoms.py:193]:
```suggestion
def test_get_reproducible_int(dcmfile:str) -> None:
```

**Review by bpinsard — CHANGES_REQUESTED** [review]
apart from missing typing, LGTM

**Review by bpinsard — APPROVED** [review]

File: heudiconv/tests/test_dicoms.py [hunk 3]
```diff
@@ -180,26 +180,17 @@ def test_get_datetime_from_dcm_wo_dt() -> None:
...
-
+    mw = dw.wrapper_from_file(dcmfile)
+    seqinfo = create_seqinfo(mw, [dcmfile], op.basename(dcmfile))
+    assert seqinfo.sequence_name
 
-def test_get_reproducible_int() -> None:
-    dcmfile = op.join(TESTS_DATA_PATH, "phantom.dcm")
 
+@pytest.mark.parametrize("dcmfile", TEST_DICOM_PATHS)
+def test_get_reproducible_int(dcmfile:str) -> None:
```
Comment (yarikoptic) [inline heudiconv/tests/test_dicoms.py:193]:
```suggestion
def test_get_reproducible_int(dcmfile: str) -> None:
```
lint

**Pull request merged**